test/evergreen/gcp_auth.json

```diff
@@ -0,0 +1,12 @@
+{
+    "type": "service_account",
+    "project_id": "axial-radius-373703",
+    "private_key_id": "f4628fbbe1227933d8909320b8711d873961205b",
+    "private_key": "privateKey",
```

Comment:
can we do "${private_key}" here instead? Would like to at least denote that is a variable that we are swapping in and out.

Reply:
I am hesitant to use "${private_key}" here as I'd need to strip the $ before calling the command as it would be interpreted as a variable to my knowledge.

Reply:
good point, lets do {private_key}?

Reply:
updated: e33a0155f8f59f6938296201e55a2f7e401a7f38